wbooks/src/redux/book/reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { AnyAction } from 'redux';
+import { BOOKS_MOCK } from '@constants/mockBooks';
+import { Book } from '@interfaces/books';
+
+import { actions } from './actions';
+
+const initialState: Book[] = BOOKS_MOCK;
```

Comment:
Check this `initialState`, each reducer must be an object
```
const initialState: BookState = {
 books: []
};
```

Create `BookState` interface. Also, add in this PR the `SUCCESS`  and `FAILURE` cases too